neuro_extras/main.py

```diff
@@ -200,6 +203,30 @@ def image_build(file: str, build_arg: Sequence[str], path: str, image_uri: str)
     run_async(_build_image(file, path, image_uri, build_arg))
 
 
+@image.command("copy")
+@click.argument("source")
+@click.argument("destination")
+def image_copy(source: str, destination: str) -> None:
+    run_async(_copy_image(source, destination))
+
+
+async def _copy_image(source: str, destination: str) -> None:
+    with tempfile.TemporaryDirectory() as tmpdir:
+        async with neuro_api.get() as client:
+            remote_image = client.parse.remote_image(image=source)
+        dockerfile_path = Path(f"{tmpdir}/Dockerfile")
+        with open(str(dockerfile_path), "w") as f:
+            f.write(
+                textwrap.dedent(
+                    f"""\
+                    FROM {_as_repo_str(remote_image)}
+                    LABEL original_url={source}
```

Comment:
should we have some name spacing: e.g. `neu.ro/source-image-uri` dunno?

Reply:
oh, i see. Good idea!